README.md

```diff
@@ -90,21 +90,36 @@ The Treasurer shall:
 - Prepare an annual budget for the group as well as budgets for specific events
 - Keep track of the blank bank checks and receipts
 
-### Project Manager
+### Resource Director
 
-The Project Manager shall:
+The Resource Director shall:
 
 - Keep track of the list of current active projects registered under the club
 - Keep track of the list of members working on the certain projects.
 - Ensure and regulate the projects and leaders working under the projects
-- Cooperate with the Resource Manager
+- Moderate communications channels.
+- Setup meeting rooms (virtual)
+- Cooperate with the Technology Chief
+
+### Technology Chief
+
+The Technology Chief shall:
+
+- Responsible for maintaining the permissions to access into the Github accounts
+- Responsible for maintaining Github configurations
+- Responsible for configuring communication channels (Discord,..)
+- Work with Project and Workshop Directors to curate resources.
+- Gain/remove permission for individual users and project leaders
+- Maintain the access and passwords for paid services and files
+- Keep track of all assets currently controlled by the club
+- Store the passwords and account names the Github, Club email, servers, Google Drive, and other paid and free services
```

Comment:
One other thing that I would say is that we should try to avoid listing specific tools and stuff in the constitution as this document will be quite difficult to change once we submit it, and these tools/tech will get outdated pretty fast. Maybe get rid of the GitHub and Discord and use more generic descriptions like "project management tools" and "communications technology", etc.